message_adapter/cumulus_message.py

```diff
@@ -41,7 +41,7 @@ def load_remote_event(event):
             remote_event = json.loads(data['Body'].read().decode('utf-8'))
             replacement_targets = parsed_json_path.find(event)
             if not replacement_targets or len(replacement_targets) != 1:
-                raise Exception(f'Remote event configuration target {target_json_path} invalid')
+                raise ValueError(f'Remote event configuration target {target_json_path} invalid')
```

Comment:
The Python update also came with some new linting defaults. This fixes an "Exception too general" rule.